### ISSUE TITLE
Fabric loader error 0.4.3

### ISSUE THREAD

**Issue by holoroom3** [FabricMC/fabric#285]
The game crashed whilst initializing game
Error: java.lang.RuntimeException: Could not execute entrypoint stage 'client' due to errors!

---- Minecraft Crash Report ----
// I'm sorry, Dave.

Time: 30/06/19 15:15
Description: Initializing game

java.lang.RuntimeException: Could not execute entrypoint stage 'client' due to errors!
	at net.fabricmc.loader.entrypoint.hooks.EntrypointUtils.logErrors(EntrypointUtils.java:45)
	at net.fabricmc.loader.entrypoint.hooks.EntrypointClient.start(EntrypointClient.java:35)
	at net.minecraft.class_310.method_1503(class_310.java)
	at net.minecraft.class_310.method_1514(class_310.java:395)
	at net.minecraft.client.main.Main.main(Main.java:154)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:170)
	at net.fabricmc.loader.launch.knot.Knot.init(Knot.java:126)
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:26)
	Suppressed: java.lang.NoClassDefFoundError: net/fabricmc/fabric/api/resource/IdentifiableResourceReloadListener
		at com.mamiyaotaru.voxelmap.fabricmod.FabricModVoxelMapBootstrapper.onInitializeClient(FabricModVoxelMapBootstrapper.java:10)
		at net.fabricmc.loader.entrypoint.hooks.EntrypointClient$$Lambda$1347/680533991.accept(Unknown Source)
		at net.fabricmc.loader.entrypoint.hooks.EntrypointUtils.lambda$logErrors$0(EntrypointUtils.java:38)
		at net.fabricmc.loader.entrypoint.hooks.EntrypointUtils$$Lambda$1346/1881258837.accept(Unknown Source)
		at java.util.ArrayList.forEach(ArrayList.java:1249)
		at net.fabricmc.loader.entrypoint.hooks.EntrypointUtils.logErrors(EntrypointUtils.java:36)
		... 11 more
	Caused by: java.lang.ClassNotFoundException: net.fabricmc.fabric.api.resource.IdentifiableResourceReloadListener
		at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
		at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
		at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
		at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
		at net.fabricmc.loader.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:160)
		at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
		... 17 more

A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Client thread
Stacktrace:
	at net.fabricmc.loader.entrypoint.hooks.EntrypointUtils.logErrors(EntrypointUtils.java:45)
	at net.fabricmc.loader.entrypoint.hooks.EntrypointClient.start(EntrypointClient.java:35)
	at net.minecraft.class_310.method_1503(class_310.java)

-- Initialization --
Details:
Stacktrace:
	at net.minecraft.class_310.method_1514(class_310.java:395)
	at net.minecraft.client.main.Main.main(Main.java:154)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:170)
	at net.fabricmc.loader.launch.knot.Knot.init(Knot.java:126)
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:26)

-- System Details --
Details:
	Minecraft Version: 1.14.2
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_51, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 1331313552 bytes (1269 MB) / 1610612736 bytes (1536 MB) up to 2147483648 bytes (2048 MB)
	JVM Flags: 8 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx2G -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M
	Launched Version: fabric-loader-0.4.3+build.134-1.14.2+build.7
	LWJGL: 3.2.1 build 12
	OpenGL: NO CONTEXT
	GL Caps: 
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fabric'
	Type: Client (map_client.txt)
	Resource Packs: ~~ERROR~~ NullPointerException: null
	Current Language: ~~ERROR~~ NullPointerException: null
	CPU: <unknown>

**Comment by modmuss50** [comment]
You need to install fabric api from https://www.curseforge.com/minecraft/mc-mods/fabric-api/files

Just drop the latest jar folder in the mods directory.

**Comment by holoroom3** [comment]
Ah thank you xDSent from my Samsung Galaxy smartphone.
-------- Original message --------From: Modmuss50 <notifications@github.com> Date: 30/06/2019  15:32  (GMT+00:00) To: FabricMC/fabric <fabric@noreply.github.com> Cc: holoroom3 <carollp61@gmail.com>, Author <author@noreply.github.com> Subject: Re: [FabricMC/fabric] Fabric loader error 0.4.3 (#285) You need to install fabric api from https://www.curseforge.com/minecraft/mc-mods/fabric-api/files
Just drop the latest jar folder in the mods directory.

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or mute the thread.
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/FabricMC/fabric/issues/285?email_source=notifications\u0026email_token=AMPTTWYLDBBU2FRCO2Z5Q6LP5C7YLA5CNFSM4H4MVBIKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODY4NLDY#issuecomment-507041167",
"url": "https://github.com/FabricMC/fabric/issues/285?email_source=notifications\u0026email_token=AMPTTWYLDBBU2FRCO2Z5Q6LP5C7YLA5CNFSM4H4MVBIKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODY4NLDY#issuecomment-507041167",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by krissy60** [comment]
I had fabric for Minecraft 1.14.2, installed which worked perfectly, however when I upgraded to Minecraft  1.14.3  and downloaded Fabric for 1.14.3  installed it and added the latest Fabric api the Minecraft launcher crashes every time select the frabric-loader-1.14.13 profile which says it is fabric-loader-0.4.8+build.155-1.14.3  Sorry I have no crash report, the button to print one is there, but it does not give me a crash report.   The fabric api I installed was 0.3.0+build 187 Was I supposed to uninstall the 1.14.2 version?  I did not because I am still playing on some server that use 1.14.2.  If I do need to uninstall the previous version, how do I do it?
